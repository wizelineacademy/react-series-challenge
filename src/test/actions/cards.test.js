import React from 'react';
import cardsActions from '../../actions/cards';

describe('Card Actions', () => {
	const payload = { };
	test('should return loadCards action', () => {
		expect(cardsActions.creators.loadCards(payload)).toMatchSnapshot();
	});
	test('should return loadCards action !payload', () => {
		expect(cardsActions.creators.loadCards()).toMatchSnapshot();
	});
	test('should return addFavorites action', () => {
		expect(cardsActions.creators.addFavorites(payload)).toMatchSnapshot();
	});
	test('should return addFavorites action !payload', () => {
		expect(cardsActions.creators.addFavorites()).toMatchSnapshot();
	});
	test('should return removeFavorites action', () => {
		expect(cardsActions.creators.removeFavorites(payload)).toMatchSnapshot();
	});
	test('should return removeFavorites action !payload', () => {
		expect(cardsActions.creators.removeFavorites()).toMatchSnapshot();
	});
	test('should return setFavorites action', () => {
		expect(cardsActions.creators.setFavorites(payload)).toMatchSnapshot();
	});
	test('should return setFavorites action !payload', () => {
		expect(cardsActions.creators.setFavorites()).toMatchSnapshot();
	});
});