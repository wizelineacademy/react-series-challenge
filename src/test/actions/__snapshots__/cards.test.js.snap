// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Actions should return addFavorites action !payload 1`] = `
Object {
  "payload": Object {},
  "type": "ADD_FAVORITES",
}
`;

exports[`Card Actions should return addFavorites action 1`] = `
Object {
  "payload": Object {},
  "type": "ADD_FAVORITES",
}
`;

exports[`Card Actions should return loadCards action !payload 1`] = `
Object {
  "payload": Object {},
  "type": "LOAD_CARDS",
}
`;

exports[`Card Actions should return loadCards action 1`] = `
Object {
  "payload": Object {},
  "type": "LOAD_CARDS",
}
`;

exports[`Card Actions should return removeFavorites action !payload 1`] = `
Object {
  "payload": Object {},
  "type": "REMOVE_FAVORITES",
}
`;

exports[`Card Actions should return removeFavorites action 1`] = `
Object {
  "payload": Object {},
  "type": "REMOVE_FAVORITES",
}
`;

exports[`Card Actions should return setFavorites action !payload 1`] = `
Object {
  "payload": Object {},
  "type": "SET_FAVORITES",
}
`;

exports[`Card Actions should return setFavorites action 1`] = `
Object {
  "payload": Object {},
  "type": "SET_FAVORITES",
}
`;
